packages/cli/src/installers/node/index.ts

```diff
@@ -144,7 +118,80 @@ export async function installNode(pkg: { [key: string]: any }) {
     )
   }
 
-  console.log(displayOutroMessage(pushApiKey, name))
+  const { method } = await inquirer.prompt([
+    {
+      type: "list",
+      name: "method",
+      message:
+        "Which method of configuring AppSignal in your project do you prefer?",
+      choices: [
+        "Using an appsignal.js configuration file.",
+        "Using system environment variables."
```

Comment:
Do we want to offer both of these methods? Have we considered always writing the file without the `pushApiKey` config option, and always telling people to either use the environment variable or add the `pushApiKey` option?

I'm thinking it's one less choice they have to make.